lib/matplotlib/transforms.py

```diff
@@ -2892,7 +2892,7 @@ def nonsingular(vmin, vmax, expander=0.001, tiny=1e-15, increasing=True):
     return vmin, vmax
 
 
-def interval_contains(interval, val):
+def interval_contains(interval, val, rtol=1e-10):
```

Comment:
I feel like so as to not break downstream users relying on this being exact, the default should be `rtol=0`

Reply:
@eric-wieser you probably understand the numerical issues better than me, but is this comparison ever "exact"?

Reply:
Yes, comparison is always exact. The danger is when you make assumptions about rounding in operations prior to the comparison, but that's for the caller to decide.

Reply:
OK, done...